server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ACVRDownloadByCounty.java

```diff
@@ -50,7 +55,7 @@ public EndpointType endpointType() {
    */
   @Override
   public String endpointName() {
-    return "/acvr/:counties";
+    return "/acvr/county/:counties";
```

Comment:
I'm happy to change that. Seems pretty straightforward to do... but what would `POST /acvr/county/11` mean then? Actually uploading an ACVR for that particular county rather than submitting the county's name as a parameter to the upload?